src/matches.ts

```diff
@@ -0,0 +1,58 @@
+import { purry } from './purry';
+
+function _matches(target: any, condition: any) {
+  if (typeof condition === 'function') return condition(target);
```

Comment:
Prefer blocks
```suggestion
  if (typeof condition === 'function') {
    return condition(target);
  }
```

Reply:
I feel this is more personal preference than anything else, but I don't care that much either way. I'll merge the suggestion.

Reply:
With auto-formatting I agree that it's less likely that someone would introduce another statement and the if would break, but it's also just a matter of consistency. Anyway, unless we turn on "curly" you are right that his is a preference... feel free to ignore